src/App.js

```diff
@@ -1,9 +1,14 @@
+import { BrowserRouter } from "react-router-dom";
+
 import './App.css';
+import PrimaryContainer from './components/PrimaryContainer';
 
 function App() {
   return (
     <div className="App">
-      <h1>Games!</h1>
+      <BrowserRouter>
+        <PrimaryContainer/>
```

Comment:
Just make sure to keep your components semantic for easy readability when others look through your code base